tests/components/test_ai_model.py

```diff
@@ -94,16 +94,20 @@ class Resume(BaseModel):
     def test_literal(self):
         @ai_model
         class LLMConference(BaseModel):
-            speakers: list[Literal["Adam", "Nate", "Jeremiah"]]
+            speakers: list[
+                Literal["Adam", "Nate", "Jeremiah", "Marvin", "Billy Bob Thornton"]
+            ]
 
         x = LLMConference(
             """
             The conference for best LLM framework will feature talks by
             Adam, Nate, Jeremiah, Marvin, and Billy Bob Thornton.
         """
         )
-        assert len(set(x.speakers)) == 3
-        assert set(x.speakers) == set(["Adam", "Nate", "Jeremiah"])
+        assert len(set(x.speakers)) == 5
+        assert set(x.speakers) == set(
+            ["Adam", "Nate", "Jeremiah", "Marvin", "Billy Bob Thornton"]
```

Comment:
hmm i was thinking i want my `Literal` to filter to only the `speakers` that I was interested in, so i purposefully left the extra person in the input out of the `Literal` to test that